src/plotman/plotman.py

```diff
@@ -309,15 +310,23 @@ def main() -> None:
                         job.suspend()
 
                         temp_files = job.get_temp_files()
-                        print('Will kill pid %d, plot id %s' % (job.proc.pid, job.plot_id))
-                        print('Will delete %d temp files' % len(temp_files))
-                        conf = input('Are you sure? ("y" to confirm): ')
+                        
+                        if args.force:
+                            conf = 'y'
+                        else:
+                            conf = input('Are you sure? ("y" to confirm): ')
+
                         if (conf != 'y'):
-                            print('canceled.  If you wish to resume the job, do so manually.')
+                            print('Canceled.  If you wish to resume the job, do so manually.')
                         else:
+                            print('Will kill pid %d, plot id %s' % (job.proc.pid, job.plot_id))
+                            print('Will delete %d temp files' % len(temp_files))
```

Comment:
Don't we still want this before the confirmation?  (and also regardless of `--force`)